### ISSUE TITLE
Hypothesis

### ISSUE THREAD

**Issue by paleomitchelljs** [paulpayne20/Tasks#1]
Hi Paul,

Your hypothesis and data are not uploaded, and most of the R code for your tasks can't work. If you need help or guidance, please come by my office hours.

**Comment by paulpayne20** [comment]
Yes, I do need to come by your office and figure out how to upload my data
into R. I’m going out town tomorrow, but if we could meet sometime next
week that would be great.

Thank you, Paul Payne
